MGS2-MC/MGS2MemoryManager.cs

```diff
@@ -220,9 +222,28 @@ private static List<int> FindNewPlayerOffsets(byte[] buffer)
                 throw new AggregateException($"Failed to find player offset: ", e);
             }
         }
+        
+        private static Process GetMgs2Process()
+        {
+            Process[] processes = Process.GetProcesses();
+            foreach (Process p in processes)
+            {
+                if(string.Equals(p.ProcessName, Constants.MGS2_PROCESS_NAME, StringComparison.CurrentCultureIgnoreCase))
+                    return Process.GetProcessById(p.Id);
+            }
+            return new Process();
```

Comment:
This is very similar to what i have over on dev right now in `MGS2Monitor` - slightly different in implementation and approach. I'm going to favor what I have in dev as it will play nicer with the rest of the program(and in general, I would prefer any process-specific methods to be in that class instead, for organization and clarity this class should be focused on performing actions on memory stored within the process managed by the Monitor class).

fwiw though, I will note that `return new Process();` is not a good practice. This method's intent is to find the Process *specific to* `METAL GEAR SOLID2.exe`, so if you don't find it you should throw an error or return a null.

Reply:
Good point, I'll make it return null.